sdk/app_cpu1/common/drv/timing_manager.h

```diff
@@ -12,28 +12,44 @@
 #define INTC_INTERRUPT_ID_0      (61) // IRQ_F2P[0:0]
 #define ISR0_PRIORITY            (0x08)
 #define ISR_RISING_EDGE          (0x3)
-#define DEFAULT_SENSOR_ENABLE    (0x00)
+#define DEFAULT_SENSOR_ENABLE    (0x0000)
 #define DEFAULT_PWM_RATIO        (0x1)
 #define LOWER_16_MASK            (0x0000FFFF)
-#define NUM_SENSORS              (6)
+#define NUM_SENSORS              (10)
 #define TIMING_MANAGER_BASE_ADDR (XPAR_AMDC_TIMING_MANAGER_0_S00_AXI_BASEADDR)
 #define UPPER_16_SHIFT           (16)
 
-typedef enum { EDDY_0 = 0, EDDY_1 = 1, EDDY_2 = 2, EDDY_3 = 3, ENCODER = 4, ADC = 5 } sensor_t;
+typedef enum {
+    AMDS_1 = 0,
+    AMDS_2 = 1,
+    AMDS_3 = 2,
+    AMDS_4 = 3,
+    EDDY_1 = 4,
+    EDDY_2 = 5,
+    EDDY_3 = 6,
+    EDDY_4 = 7,
+    ENCODER = 8,
+    ADC = 9
+} sensor_t;
+
+typedef enum { TM_MANUAL = 0, TM_AUTOMATIC = 1 } trigger_mode_t;
```

Comment:
FYI---often, convention for typedef of enum is a suffix of `_e`, and `_t` is used for struct typedefs.

But, this is probably fine

Reply:
Fixed anyway